uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -337,28 +338,6 @@ private static function savePublicHolidaysBalanceChanges(
         'source_type' => LeaveBalanceChange::SOURCE_ENTITLEMENT,
         'amount'      => count($publicHolidays)
       ]);
```

Comment:
The method description mentions that it creates one leave request for each public holiday, but this is not the case anymore. Please update it